test-server/server.js

```diff
@@ -13,8 +13,14 @@ app.get('/', (req, res) => {
   res.send(`${htmlIntro}<body>Testing</body>${htmlOutro}`);
 });
 
-app.get('/asset-paths', (req, res) => {
-  res.sendFile(path.join(__dirname, 'fixtures/asset-paths.html'));
+// Asset path pages
+app.get('/asset-paths/:page', (req, res) => {
```

Comment:
Moving this up above the image handlers could prevent those from working. If any are shaped like `/asset-paths/image.jpg`, this handler will match first.